course/templatetags/event_grade.py

```diff
@@ -0,0 +1,20 @@
+from django import template
```

Comment:
why is this template tag is in the course app if you are using it in the canvas app?